server.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+// Environment vars
+const port = process.env.PORT || 8080;
+
+// npm packages
+const express = require('express');
+
+const app = express();
+
+app.use(express.static(`${__dirname}/build`));
```

Comment:
Good job creating your own static server :)
what happens if the client requests an asset that is not there,
in this file its a good idea to have a cattch all after the static server to redirect to a 404 page with a link back to the home page :)

redirect to /#!/404